exercises/dnd-character/lib/dnd_character.ex

```diff
@@ -0,0 +1,15 @@
+defmodule DndCharacter do
+  @spec modifier(pos_integer) :: integer
+  def modifier(score) do
+    # code here
```

Comment:
I don't think we usually put anything inside the the stub functions and just keep it blank.